### ISSUE TITLE
is it possible to autocomplete ?

### ISSUE THREAD

**Issue by muhilham** [brianlow/FileRename#4]
Very nice plugin.
i know i can move a file into some folder inside root folder using this plugin. is there any way i can do autocomplete for folder name?

**Comment by math2001** [comment]
Yes, but not with this plugin. It might be a bit harsh from me to do this, but since it's been really long since you've raise this issue, and no one has answered...

I've done a plugin called [FileManager] that has this functionality (rename), and has the auto completions for folder

[FileManager]: https://packagecontrol.io/packages/FileManager
